modules/cnv-importing-vm-datavolume.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes.adoc
+
+[id="cnv-importing-vm-datavolume_{context}"]
+= Importing a virtual machine image into a {ProductName} object with DataVolumes
+
+To create a virtual machine from an imported image, specify the image location
+in the `VirtualMachine` configuration file before you create the virtual machine.
+
+.Prerequisites
+
+* The {product-title} Command-line Interface (CLI), commonly known as `oc`
+* A virtual machine disk image, in RAW, ISO, or QCOW2 format, optionally
+compressed by using `xz` or `gz`
+* An `HTTP` or `s3` endpoint where the image is hosted, and authentication
```

Comment:
Please remove S3.  It is not well tested nor intended for use by customers at this time.